src/pages/content/[page].js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import Admin from 'src/layout/Admin';
+import { withRouter } from 'next/router';
 
-export const Home = () => {
-  return (
-    <Admin pageTitle={''}>
-      <p>any page content</p>
+import TestingFacilityLocator from 'src/components/TestingFacilityLocator';
+
+const pages = {
+  'where-to-help': (
```

Comment:
we should probably make this "where to get help". and 'where-to-get-help', please make sure that other links do link to this page correctly.